src/Symfony/Component/Cache/Traits/Item.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Symfony\Contracts\Cache\ItemInterface;
+
+/**
+ * A short namespace-less class to serialize items with metadata.
+ *
+ * @author Nicolas Grekas <p@tchwork.com>
+ *
+ * @internal
+ */
+class Ͼ
+{
+    public const METADATA_EXPIRY_OFFSET = 1527506807;
```

Comment:
Now it's public. 
Do we need this magic number for BC?

Reply:
public on an `@internal` class, so "private" in practice
it's needed for BC, but also to pack expiry into 32bits with a longer term upper boundary

Reply:
now private